monkey/monkey_island/cc/ui/src/styles/components/HideInput.scss

```diff
@@ -0,0 +1,3 @@
+.eye-button{
+  padding: 5px !important;
```

Comment:
Does this really need the `!important`?

Reply:
I don't see this class being used anywhere.

Reply:
Deleted.